automated_analysis.py

```diff
@@ -621,35 +621,36 @@ def set_survey_percentages(survey_counts, total_survey_counts):
         paths_to_upload = glob(f"{output_dir}/*.csv")
         for i, path in enumerate(paths_to_upload):
             log.info(f"Uploading CSV {i + 1}/{len(paths_to_upload)}: {path}...")
-            drive_client_wrapper.update_or_create(
-                path, pipeline_configuration.drive_upload.automated_analysis_dir, target_folder_is_shared_with_me=True
+            drive_client_wrapper.update_or_create_batch(
+                path, pipeline_configuration.drive_upload.automated_analysis_dir, target_folder_is_shared_with_me=True,
```

Comment:
This isn't a batch upload if you're only passing in a single path.

Reply:
updated thanks.